src/MySql/docs/Update-AzMySqlFlexibleServerFirewallRule.md

```diff
@@ -0,0 +1,303 @@
+---
+external help file:
+Module Name: Az.MySql
+online version: https://docs.microsoft.com/en-us/powershell/module/az.mysql/update-azmysqlflexibleserverfirewallrule
+schema: 2.0.0
+---
+
+# Update-AzMySqlFlexibleServerFirewallRule
+
+## SYNOPSIS
+Creates a new firewall rule or updates an existing firewall rule.
+
+## SYNTAX
+
+### UpdateExpanded (Default)
+```
+Update-AzMySqlFlexibleServerFirewallRule -Name <String> -ResourceGroupName <String> -ServerName <String>
+ -EndIPAddress <String> -StartIPAddress <String> [-SubscriptionId <String>] [-DefaultProfile <PSObject>]
+ [-AsJob] [-NoWait] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### ClientIPAddress
+```
+Update-AzMySqlFlexibleServerFirewallRule -Name <String> -ResourceGroupName <String> -ServerName <String>
+ -ClientIPAddress <String> [-SubscriptionId <String>] [-DefaultProfile <PSObject>] [-AsJob] [-NoWait]
+ [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### ClientIPAddressViaIdentity
+```
+Update-AzMySqlFlexibleServerFirewallRule -InputObject <IMySqlIdentity> -ClientIPAddress <String>
+ [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### UpdateViaIdentityExpanded
+```
+Update-AzMySqlFlexibleServerFirewallRule -InputObject <IMySqlIdentity> -EndIPAddress <String>
+ -StartIPAddress <String> [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates a new firewall rule or updates an existing firewall rule.
+
+## EXAMPLES
+
+### -------------------------- EXAMPLE 1 --------------------------
+```powershell
+Update-AzMySqlFlexibleServerFirewallRule -Name rule -ResourceGroupName PowershellMySqlTest -ServerName mysql-test -EndIPAddress 0.0.0.3 -StartIPAddress 0.0.0.2
+```
```

Comment:
output should be included in the quotation marks

Reply:
The example files I wrote includes the output in the quotation marks, but the autogenerated docs file don't include it. This was the autogeneration problem I was asking you about.. Not just flexible server documents but also single server documents. Do you think the powershell version would be the problem? I use 7.0.3.

Reply:
> The example files I wrote includes the output in the quotation marks, but the autogenerated docs file don't include it. This was the autogeneration problem I was asking you about.. Not just flexible server documents but also single server documents. Do you think the powershell version would be the problem? I'm using PSVersion 5.1.19041.610 and pwsh 7.0.3..

The commit 19ad16d4ec584fe5791b06b57deb416bcc56dcbe include the codes I generated using your config. I am using the powershell core 7.0.3 but cannot reproduce your issue.